### PR TITLE
make the secondary segment at the bottom of form clickable

### LINKED ISSUE #521: Bottom segment of login and register form not completely clickable.
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
register

The bottom segment saying 'Already have an account. Log in.' is having a pointer for whole segment but is only clickable on the text part.
Similar behavior is for the login form.

**Expected**
The whole secondary segment part should be clickable.

### PR DESCRIPTION
**Pull request by sumedh123**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #521 

@niranjan94 @geekyd @CosmicCoder96 Please review.
https://oefrontend2017.herokuapp.com

### CODE REVIEW

File: app/templates/components/forms/login-form.hbs [hunk 1]
```diff
@@ -35,9 +35,13 @@
             <a href="{{href-to 'forgot-password'}}" class="text muted"> {{t 'Forgot your password ?'}}</a>
           </div>
         </div>
-        <div class="ui secondary segment center aligned">
-          <a href="{{href-to 'register'}}" class="text muted weight-800">{{t 'New to site? Create Account'}}</a>
-        </div>
+        <a href="{{href-to 'register'}}">
```
Comment (niranjan94) [inline app/templates/components/forms/login-form.hbs]: This is incorrect semantic markup. `segment` should be a direct child of `segments`

File: app/templates/components/forms/register-form.hbs [hunk 2]
```diff
@@ -33,8 +33,12 @@
         </button>
       </div>
     </div>
-    <div class="ui secondary segment center aligned">
-      <a href="{{href-to 'login'}}" class="text muted weight-800">{{t 'Already have an account? Login'}}</a>
-    </div>
+    <a href="{{href-to 'login'}}">
```
Comment (niranjan94) [inline app/templates/components/forms/register-form.hbs]: This is incorrect semantic markup. `segment` should be a direct child of `segments`

**Comment by sumedh123** [comment]
@niranjan94 Tests are passing locally. But here they are failing due to crashing of phantom. Please review I have made the changes.

**Comment by niranjan94** [comment]
@sumedh123 update your branch. I have configured travis to run tests with Chrome instead.

File: app/templates/components/forms/login-form.hbs [hunk 3]
```diff
@@ -35,9 +35,13 @@
             <a href="{{href-to 'forgot-password'}}" class="text muted"> {{t 'Forgot your password ?'}}</a>
           </div>
         </div>
-        <div class="ui secondary segment center aligned">
-          <a href="{{href-to 'register'}}" class="text muted weight-800">{{t 'New to site? Create Account'}}</a>
-        </div>
+        {{#link-to 'register'}}
```
Comment (niranjan94) [inline app/templates/components/forms/login-form.hbs]: Like I said before, a `segment` has to be the direct child of `segments`. Using link-to doesn't change that.

**Comment by sumedh123** [comment]
@niranjan94 The only way to do this would be setting an action on the div then since I cannot wrap the whole thing in an anchor tag. It disturbs the design

**Comment by niranjan94** [comment]
@sumedh123 just add the segment classes directly to the anchor tag, and add the following to `overrides.scss`

```scss
a.ui.segment {
  display: block !important;
}
```

Simple.

**Comment by niranjan94** [comment]
An anchor tag by default is an inline element. But segments have to be block elements.

**Comment by sumedh123** [comment]
@niranjan94 Please review. https://oefrontend2017.herokuapp.com

File: app/templates/components/forms/login-form.hbs [hunk 4]
```diff
@@ -35,9 +35,11 @@
             <a href="{{href-to 'forgot-password'}}" class="text muted"> {{t 'Forgot your password ?'}}</a>
           </div>
         </div>
-        <div class="ui secondary segment center aligned">
-          <a href="{{href-to 'register'}}" class="text muted weight-800">{{t 'New to site? Create Account'}}</a>
-        </div>
+        <a href="{{href-to 'register'}}" class="ui secondary segment center aligned">
+          <div class="text muted weight-800">
```
Comment (niranjan94) [inline app/templates/components/forms/login-form.hbs]: Why is this div even needed? The classes can be added to the anchor element above.

**Comment by sumedh123** [comment]
@niranjan94 Corrected. Please review. https://oefrontend2017.herokuapp.com

**Pull request merged**